task_queue.rb

```diff
@@ -1,6 +1,12 @@
-require "set"
-require_relative "task"
-require_relative "queue_worker"
+# frozen_string_literal: true
+
+require 'set'
```

Comment:
we actually decided to just use double quotes " in all the places instead of single quotes '